dts/rules.mk

```diff
@@ -4,5 +4,6 @@ ifeq ($(PROJECT), msm8916-secondary)
 DTBS += \
 	$(LOCAL_DIR)/apq8016-samsung.dtb \
 	$(LOCAL_DIR)/msm8916-longcheer-l8150.dtb \
-	$(LOCAL_DIR)/msm8916-samsung.dtb
+	$(LOCAL_DIR)/msm8916-samsung.dtb \
+	$(LOCAL_DIR)/msm8916-motorola-harpia.dtb
```

Comment:
keep this alphabetically ordered, i.e. add `msm8916-motorola-harpia.dtb` before `msm8916-samsung`